DotNet/proxy.xsd

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.1.1beta" encoding="utf-8"?>
```

Comment:
@jgravois - I still see `xml version="1.1.1beta"` instead of `xml version="1.0"`.